ApollosProject/apollos-apps
# chore: Remove Share Post from Community

## ✏️ Description
This PR removes the "share" button that was showing up on individual Community Posts. We don't have the "post single" pages hooked up with deep links, yet, so we're gonna remove it until we can get this worked out in the future.

## 🔬 To Test
1. Open up Communities
2. Make sure that you don't see any share buttons showing up

## 📸 Screenshots
| Before | After |
| --- | --- |
| ![CleanShot_2023 02 28__000013](https://user-images.githubusercontent.com/45076058/221986881-ca20fccc-42cb-488d-b57e-ce2de7114895.png) | ![CleanShot_2023 02 28__000014](https://user-images.githubusercontent.com/45076058/221986916-3a0426db-fc42-4f44-b711-fbd46d03200c.png)


## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
